Packs/SuspiciousDomainHunting/Layouts/layoutscontainer-Suspicious_Domain_Hunting_Incident_Layout.json

```diff
@@ -0,0 +1,1592 @@
+                                        "simple": "Dear ${Domain.Whois.Registrar.Name.[0]},\n\nI am writing to report abusive activity on the domain ${incident.domain}. \nThis activity is in violation of your terms of service or acceptable use policy.\nI kindly request your prompt investigation and necessary action to address this abuse, which may include suspending the domain.\n\nYour swift response and cooperation in resolving this matter are greatly appreciated.\n\nSincerely,\n${inputs.contactEmail}"
+                                    },
+                                    "subject": {
+                                        "simple": "Report of Abuse ${incident.domain}"
+                                    },
+                                    "to": {
+                                        "simple": "oitzhak@paloaltonetworks.com"
+                                    }
+                                },
+                                "buttonClass": "error",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "cff5b9c0-7956-11ee-98e4-c5340a9b9a3f",
+                                "index": 3,
+                                "name": "Ask For Takedown",
+                                "scriptId": "EWSO365|||send-mail",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Quick Actions",
+                        "static": false,
+                        "w": 1,
+                        "x": 2,
+                        "y": 4
+                    },
+                    {
+                        "h": 4,
+                        "i": "caseinfoid-jk8ghehhzw-field-changed-caseinfoid-field-changed-caseinfoid-cf93f860-7f01-11ee-b497-af0cc533d63c",
+                        "items": [],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Rasterize",
+                        "query": "82f70396-4be3-4e57-838b-22f614e83415",
+                        "queryType": "script",
+                        "static": false,
+                        "type": "dynamic",
+                        "w": 1,
+                        "x": 2,
+                        "y": 0
+                    },
+                    {
+                        "h": 1,
+                        "i": "caseinfoid-jk8ghehhzw-field-changed-caseinfoid-3af909a0-816a-11ee-8640-1f7a9f394d1e",
+                        "items": [],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Verdict Results",
```

Comment:
Please ensure that the layout utilizes the verdict result script that already exists in the commonScripts.